util/OptionsDB.cpp

```diff
@@ -431,6 +431,24 @@ void OptionsDB::SetFromCommandLine(const std::vector<std::string>& args) {
     }
 }
 
+void OptionsDB::SetFromFile(const boost::filesystem::path& file_path,
+                            const std::string& version) {
```

Comment:
For multi-line function definitions, the { should always be on a line of its own.

You can put the { at the end of the definition line if it fits in one line, but not if it's split over multiple lines.

Applies for if, loop, and function definition blocks.